src/status_im/contexts/communities/actions/community_options/view.cljs

```diff
@@ -137,13 +137,11 @@
 
 (defn not-joined-options
   [id token-gated? intro-message]
-  [[(when-not token-gated? (view-members id))
-    (when-not token-gated? (view-rules id intro-message))
-    (mark-as-read id)
-    (invite-contacts id)
-    (when token-gated? (view-token-gating id))
-    (show-qr id)
-    (share-community id)]])
+  (let [common   [(show-qr id) (share-community id)]
+        specific (if token-gated?
+                   [(invite-contacts id) (view-token-gating id)]
+                   [(view-members id) (view-rules id intro-message) (invite-contacts id)])]
+    [(concat specific common)]))
```

Comment:
When the user isn't joined we also showed the "Mark as read" button, but it's was wrong, here I'm also fixing it